esmvaltool/cmorizers/obs/cmorize_obs_cowtanway.py

```diff
@@ -0,0 +1,102 @@
+"""ESMValTool CMORizer for CowtanWay.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://www-users.york.ac.uk/~kdc3/papers/coverage2013/series.html
+
+Last access
+    20200226
+
+Download and processing instructions
+    Download the following files:
+        '{version}_0_0.nc.gz'
```

Comment:
There are a lot of links in this page, it would be simpler if you could add the actual link to the files to be downloaded.

Reply:
Done.